hawkbit-mgmt-resource/src/main/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtTargetMapper.java

```diff
@@ -168,7 +148,7 @@ public static MgmtTarget toResponse(final Target target) {
             targetRest.setInstalledAt(installationDate);
         }
 
-        targetRest.add(linkTo(methodOn(MgmtTargetRestApi.class).getTarget(target.getControllerId())).withRel("self"));
+        targetRest.add(linkTo(methodOn(MgmtTargetResource.class).getTarget(target.getControllerId())).withSelfRel());
```

Comment:
Why are you using the `MgmtTargetResource.class` here and not the `MgmtTargetRestApi.class`?